packages/babel-types/src/constants.js

```diff
@@ -43,7 +43,7 @@ export const BINARY_OPERATORS = [
 ];
 
 export const BOOLEAN_UNARY_OPERATORS = ["delete", "!"];
-export const NUMBER_UNARY_OPERATORS = ["+", "-", "++", "--", "~"];
```

Comment:
That's why were in alphas. I'd be surprised if anyone actually used this, though.